### DISCUSSION TITLE
Toggle route states in devtools?

### DISCUSSION THREAD

**Discussion by mrlubos** [TanStack/router#1895] [Ideas]
Correct me if I'm wrong, but there's currently no way to do this, right? I'd like to suspend routes from devtools to test various states, i.e. render `errorComponent`, `pendingComponent`, `notFoundComponent`, and of course `component`.

**Comment by DennisJosefsson** [comment]
This is something I'd like as well. It's possible to toggle suspense and error in the React dev tool extension, but you have to find your component in the dom tree first and the TanStack router dev tool gives a much better overview of the relevant routes.
